ExtractorLauncher/ExtractorStarter.cs

```diff
@@ -207,6 +207,12 @@ public static class ExtractorStarter
                 return "datapoints-batch must be greater than 0";
             }
 
+            if (config?.Cognite?.MetadataTargets?.Clean?.MetadataAsJson == true && string.IsNullOrWhiteSpace(config.Cognite.MetadataTargets.Clean.Space))
```

Comment:
This if-condition has two parts, where it is accessing inside several levels of properties, but it is using null-conditional operator only for the first part. 
It may just be coincidence that they are in the same parts, so if the first part succeeds, where all properties are non-null, the second part will also suceed.

Would have been safe to use null-conditions also for the second part (the check for Space).